actors/market/tests/on_miner_sectors_terminate.rs

```diff
@@ -323,97 +342,3 @@ fn fail_when_caller_is_not_a_storage_miner_actor() {
 
     check_state(&rt);
 }
-
-// Converted from: https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/market/market_test.go#L1448
-#[test]
-fn fail_when_caller_is_not_the_provider_of_the_deal() {
```

Comment:
These three tests cases still seem relevant and I think should still have tests after this change.

Reply:
`fail_when_caller_is_not_the_provider_of_the_deal`: this is impossible now since the caller does not specify the deals, they are looked up. Only inconsistent internal state from activation could cause this. Specifying a non-existent sector is already tested.

`fail_when_deal_has_been_published_but_not_activated`: impossible since deal IDs are not specified by the caller. Only activated deals exist in the provider_sectors map. Specifying a sector with an already-expired deal is tested.

`termination_of_all_deals_should_fail_when_one_deal_fails`: impossible to specify a deal that can fail to be terminated, e.g. because not activated.

These removed tests reflect fundamental simplifications we gained from decoupling.

Reply:
Nice, thanks for explaining